### ISSUE TITLE
Current ci fails for new pr

### ISSUE THREAD

**Issue by enricosada** [dotnet/cli#4840]
CI is green, but new pr fails (with no changes)

## Steps to reproduce

Send a pr with empty commit.

## Expected  behavior

CI: Green

## Actual behavior

- Intermittent #4839 test failure
- always Test failure .VSTestFailTest (from MSTestSuite)

The current `HEAD` eb8e0cfa409d2ef89091f2058487be0e48e8a32e has a test failure  [.VSTestFailTest (from MSTestSuite)](https://ci.dot.net/job/dotnet_cli/job/rel_1.0.0/job/release_windows_nt_x64_prtest/119/testReport/(root)/(empty)/VSTestFailTest/).

<details>
  <summary>test failure details</summary>

```
Failing for the past 4 builds (Since Fallita#116 )
Took 52 ms.
Stacktrace

                                        MESSAGE:
                                        Assert.Fail failed. 
                                        +++++++++++++++++++
                                        STACK TRACE:
                                           at TestNamespace.VSTestTests.VSTestFailTest() in D:\j\workspace\release_windo---4c3b899b\test\dotnet-test.Tests\bin\Release\netcoreapp1.0\TestWillCreateTrxLogger\VSTestDotNetCore\Tests.cs:line 17
```
</details>

## Environment data

latest `HEAD` https://github.com/dotnet/cli/commit/eb8e0cfa409d2ef89091f2058487be0e48e8a32e

**Comment by enricosada** [comment]
fixed by #4838 /cc @livarcocc @piotrpMSFT

**Comment by piotrpMSFT** [comment]
#4838 is merged.
